source/developers-guide/debugging/index.md

```diff
@@ -148,6 +148,7 @@ As you can restrict the plugin to your own IP address, this is also suitable for
 Debugging ExtJS errors during development can be very time consuming. Errors like `c is not a constructor` are often not helpful. To address this, you can include `ext-all-debug.js` instead of the default `ext-all.js` file:
 Edit the themes/Backend/ExtJs/backend/base/header.tpl` file `and replace `ext-all.js` with `ext-all-debug.js` in the block `backend/base/header/javascript`. 
 After clearing the cache and reloading the backend, the "debug" ExtJS file is included, which displays more helpful error messages.
+This extension for Chrome might also be useful: [App Inspector for Sencha™](https://chrome.google.com/webstore/detail/app-inspector-for-sencha/pbeapidedgdpniokbedbfbaacglkceae)
```

Comment:
I would put this sentence into another block. So just make a new line before this sentence. Also I would rephrase this to:

The Chrome extension [App Inspector for Sencha™](https://chrome.google.com/webstore/detail/app-inspector-for-sencha/pbeapidedgdpniokbedbfbaacglkceae) might also be very useful.